crates/bootstrap_client/src/lib.rs

```diff
@@ -0,0 +1,49 @@
+//! A client for the Kitsune2 bootstrap server.
+
+#![deny(missing_docs)]
+
+use kitsune2_api::{AgentInfoSigned, DynVerifier, K2Error, K2Result, SpaceId};
+use std::sync::Arc;
+
+/// Send the agent info, for the given space, to the bootstrap server.
+pub fn put(
```

Comment:
```suggestion
pub fn blocking_put(
```

Do we want to call out explicitly that these are not async functions to help us remember we need to wrap them in spawn_blockings later?

Reply:
Done: https://github.com/holochain/kitsune2/pull/129/commits/531a406292a63bb0d27852900ca937266eec7d21